lib/repl/repl.js

```diff
@@ -266,7 +266,24 @@ function displayUsageFor(name) {
     return;
   }
   console.log();
+  if (e.deprecated) {
+    console.log(`DEPRECATED ${desc(e.deprecated)}`);
+    console.log();
```

Comment:
Any specific reason for empty `console.log`?
If it is just to achieve a new line, consider using '\n'. If worried about os specific new lines consider using `require('os').EOL`.

Reply:
I was reusing what was previously in place. `EOL` is a better fit as suggested.